README.md

```diff
@@ -10,6 +10,14 @@ It will eventually have MIDI input, and a way to share music and instruments wit
 
 [Test it out here!](https://fazeoid.netlify.app/)
 
+Supported browsers:
+ - Chrome desktop
+ - Chrome mobile
+ - Firefox desktop
+
```

Comment:
I think we should add a section of "currently unsupported browsers" if we have supported browsers. This would include:

- Firefox Android
- Safari (Desktop & mobile, and every ios browser)